api/events.go

```diff
@@ -0,0 +1,105 @@
+package api
+
+import (
+	"time"
+
+	"go.sia.tech/core/types"
+	"go.sia.tech/renterd/webhooks"
+)
+
+const (
+	ModuleContractSet = "contract_set"
+	ModuleConsensus   = "consensus"
+	ModuleContract    = "contract"
```

Comment:
Plural? I originally had it that way but I think with the events being "update" it makes sense to keep it singular.